### PR TITLE
Remove QSI from Tech Tree

### PR DESCRIPTION
**Pull request by SlamBamActionman**
## About the PR

Removes the Quantum Spin Inverter from the Science tech tree, and the honki shuttle. It's only available via admin entity spawn menu.

## Why / Balance

Requested by Game Masters due to bugs.
Note that this makes the Experimental T3 techs only have one option. 

## Technical details

This only comments out the code for where the QSI is accessed in the game, not the prototype itself. 

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

- remove: Removed QSI

### CODE REVIEW

File: Resources/Maps/Shuttles/ShuttleEvent/honki.yml
Comment (TsjipTsjip) [inline Resources/Maps/Shuttles/ShuttleEvent/honki.yml:1]: This should be done through a migration (`Resources/migrations.yml`) instead.

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]

File: Resources/Maps/Shuttles/ShuttleEvent/honki.yml
Comment (SlamBamActionman) [reply Resources/Maps/Shuttles/ShuttleEvent/honki.yml:1]: I've tested this now and unfortunately this does not work with `migrations.yml`. 
The issue is that the Quantum Spin Inverters are inside of a container. If the QSIs are migrated to null, the containers still have the ID entity reference, making them contain a "null object" which throws an error.

With your blessing I can change the QSIs to AI Toys or something similar, though.

Comment (TsjipTsjip) [reply Resources/Maps/Shuttles/ShuttleEvent/honki.yml:1]: Probably more trouble than it's worth.... Revert the clown shuttle file, as thought NT has stopped QSI research, others could still have a pair realistically.

Good to merge after.

**Review by TsjipTsjip — APPROVED** [review]
Bye bye QSI.

**Pull request merged**